rfc-032-keep-support-docs-with-code.md

```diff
@@ -0,0 +1,33 @@
+# Keep support documentation with code
+
+## Summary
+
+Support and other technical documentation should be kept with the code it documents, so that it can be found easily.
+
+## Problem
+
+Support documentation exists in multiple places currently, and can be confusing. Some projects have `doc` directories, some don't, and some have multiple sources of truth. We should aim to standardise this.
+
+As well as making documentation easier to find, having it in standard locations makes future automation of support documentation easier.
+
+## Proposal
+
+All source code repositories MUST have a `README.md` file in the root. This file MUST include a link to support documentation under a `Support` heading.
```

Comment:
I don't think this applies to all repositories. It's only ones that we have a support contract for.